src/core/agent/services/ipexCommunicationService.test.ts

```diff
@@ -892,41 +924,111 @@ describe("Ipex communication service of agent", () => {
         },
       },
     ]);
-    credentialStorage.getCredentialMetadata = jest.fn().mockResolvedValue({
-      id: "id",
-    });
+    credentialStorage.getCredentialMetadata = jest.fn().mockResolvedValue(null);
+    multisigService.multisigAdmit = jest
+      .fn()
+      .mockResolvedValue({ op: { name: "opName", done: true } });
+
+    notificationStorage.findAllByQuery = jest.fn().mockResolvedValue([
+      {
+        type: "NotificationRecord",
+        id: id,
+        createdAt: new Date(),
+        a: {
+          r: "/exn/ipex/grant",
+          d: "EIDUavcmyHBseNZAdAHR3SF8QMfX1kSJ3Ct0OqS0-HCW",
+        },
+        route: "/exn/ipex/grant",
+        read: true,
+        linkedGroupRequests: {
+          "EDm8iNyZ9I3P93jb0lFtL6DJD-4Mtd2zw1ADFOoEQAqw": false,
+        },
+        connectionId: "EEFjBBDcUM2IWpNF7OclCme_bE76yKE3hzULLzTOFE8E",
+        updatedAt: new Date(),
+      },
+    ]);
+
+    await ipexCommunicationService.acceptAcdc(id);
```

Comment:
We need to check here and below that we DON'T delete the notification record because it's multi-sig